packages/pangea-sdk/pangea/services/audit/models.py

```diff
@@ -441,7 +441,10 @@ def __repr__(self):
 
 
 class DownloadRequest(APIRequestModel):
-    result_id: str
+    request_id: Optional[str] = None
+    """ID returned by the search API."""
+
+    result_id: Optional[str] = None
     """ID returned by the search API."""
```

Comment:
Should they have different descriptions?

Reply:
They should but I promise when I wrote these, that's how they were in the API docs. Now I see the docs have been updated (it now clarifies which comes from the search API and which comes from the export API), so I've updated this too now.